src/theory/arith/linear/normal_form.h

```diff
@@ -266,13 +268,12 @@ class Variable : public NodeWrapper {
  }
 
   static bool isLeafMember(Node n);
-  static bool isIAndMember(Node n);
-  static bool isPow2Member(Node n);
   static bool isDivMember(Node n);
   bool isDivLike() const{
     return isDivMember(getNode());
   }
-  static bool isTranscendentalMember(Node n);
+  /** Return true if all direct children of n are polynomial members */
```

Comment:
Can you clarify here what is a polynomial member? In particular, this seems to replace a check of non-linear operators.